prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs

```diff
@@ -8,30 +8,31 @@ namespace Prime.Configuration.Database
+                    new EmailTemplate { Id = 2,  EmailType = EmailTemplateType.CommunityPracticeNotification,          Template = "<style> .underline { text-decoration: underline; } .list-item-mb { margin-bottom: 0.75rem; } </style>To: PharmaNet administrator (person responsible for coordinating PharmaNet access):<br><br>@Model.EnrolleeFullName has been approved for access to PharmaNet.<br><br><strong> To set up their access, you must forward this notification and the information below to your <span class=\"underline\">whoever sets up PharmaNet user accounts. This is usually your PharmaNet software vendor but can also be someone like an IT department or authorized individual</span>. </strong><br><br><ol><li class=\"list-item-mb\"> Name of the community health practice: </li><li class=\"list-item-mb\"> Practice address: </li><li class=\"list-item-mb\"> PharmaNet site ID, if you have it:</li><li class=\"list-item-mb\"> If @Model.EnrolleeFullName accesses PharmaNet on behalf of another user, please ensure your PharmaNet software vendor knows who the other users are, and those users enrol in PRIME.<br><br></li></ol> Your software vendor can find @Model.EnrolleeFullName's details by following this link: <a href=\"@Model.TokenUrl\">@Model.TokenUrl</a> <strong>This link will expire in 10 days.</strong><br><br>Thank you,<br><br>PRIME Support <br>1-844-397-7463<br><a href=\"mailto:PRIMEsupport@gov.bc.ca\" target=\"_top\">PRIMEsupport@gov.bc.ca</a>", ModifiedDate = MultipleEmailContentUpdateNov30th2022, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = MultipleEmailContentUpdateNov30th2022},
```

Comment:
Comparing to the content in JIRA ticket, there is an extra 'your' here.
![image](https://user-images.githubusercontent.com/108699279/206057157-782a563b-8b1d-40fa-8dfc-580b022c2342.png)

Reply:
Good catch! ... how did that happen 🤔

Reply:
Updated template

Reply:
confirmed that it is fixed.
![image](https://user-images.githubusercontent.com/108699279/206243143-abb98773-a20b-49fc-960b-686a38c9109b.png)